runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3054,7 +3054,14 @@ gcReinitializeDefaultsForRestore(J9VMThread* vmThread)
 	bool result = true;
 
 	PORT_ACCESS_FROM_JAVAVM(vm);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
 
+	extensions->usablePhysicalMemory = omrsysinfo_get_addressable_physical_memory();
+	uintptr_t potentialSoftMx = extensions->computeDefaultMaxHeapForJava(false);
+	/* if snapshot machine's memory is higher than the restore machine */
+	if(extensions->memoryMax > potentialSoftMx){
```

Comment:
conding standard: space between 'if' and '('

Reply:
Fixed, thanks